docker/tests/os.Dockerfile

```diff
@@ -63,15 +91,27 @@ RUN set -euxv; \
           binutils \
           # A realpath that works with non-existing files.
           coreutils \
-          git ca-certificates; \
+          git ca-certificates \
+          curl tar unzip; \
     elif command -v slackpkg; then \
+      # Shhhhh! Make wget quiet
+      echo "quiet = on" >> /etc/wgetrc; \
       slackpkg update; \
       # Is there a "right" way to do this?
                              # xxd for unit tests
       yes | slackpkg install vim \
                              # nm for lwhich
                              binutils \
-                             git ca-certificates; \
+                             git perl ca-certificates \
+                             # unzip
+                             infozip \
+                             # needed for update-ca-certificates
+                             perl \
+                             # libcurl -> git won't work without this
+                             cyrus-sasl; \
+                             # curl cyrus-sasl
```

Comment:
delete?

Reply:
actually, where is curl installed for slack

Reply:
It's not, slackware is wget based,